src/linearoperators/LBFGS.jl

```diff
@@ -58,49 +76,49 @@ function update!(L::LBFGS{R, T, M}, x::T, x_prev::T, gradx::T, gradx_prev::T) wh
 		L.ys_M[L.curridx] = ys
 		blockcopy!(L.s_M[L.curridx], L.s)
 		blockcopy!(L.y_M[L.curridx], L.y)
-		yty = real(vecdot(L.y, L.y))
```

Comment:
test passed also with `vecdot` but I guess it is more safe to use our `blockvecdot` no?